### DISCUSSION TITLE
Filter row if value already exist in other cell of column

### DISCUSSION THREAD

**Discussion by toog** [fbaligand/kibana-enhanced-table#267] [Q&A]
Hello

I have document with field ID and STATE
example : 
{ @timestamp: T0, ID : toto, State: busy}
{ @timestamp: T+X, ID : toto, State: ko}

I want to have a split table by STATE and show the event T0 in the first Table and at T+X in the second part of the table only

![image](https://user-images.githubusercontent.com/40697794/163133476-5b336c01-8ce7-43b4-8744-2c15622c6c59.png)

and Next

![image](https://user-images.githubusercontent.com/40697794/163133670-b4921764-72d7-48f6-ba05-93a5822ca4a4.png)

What i have today is like

![image](https://user-images.githubusercontent.com/40697794/163133831-994f1da5-60af-429f-a5fe-e7d0507dbb5c.png)

How can i used the row filter to search if value already exist in other part of table ? Or other solution ?

thank you in advance for the answer

**Comment by fbaligand** [comment]
Well, as its name says, row filter does only row filtering based on the different fields of the row. 
you can’t reference another row field.

**Comment by toog** [comment]
ok :-( maybe an idea for a new evolution ?
